config/config.go

```diff
@@ -248,3 +239,14 @@ func ParseConfigFlag() string {
 
 	return *configPath
 }
+
+const (
+	FreePlanExpiryTag = "free-plan-expiry"
```

Comment:
❓ Can we use another tag (e.g. `default-expiry`) or is the logic tied to this constant? I think it'd be better if we could  avoid using a cloud-only term like "plan"

Reply:
updated and removed messages in PR description